features/src/autogluon/features/generators/datetime.py

```diff
@@ -53,11 +59,17 @@ def _compute_fillna_map(self, X: DataFrame):
             fillna_datetime = datetime_series.mean()
             fillna_map[datetime_feature] = fillna_datetime
         return fillna_map
+    
+    def _get_country(self): # Obtain the country used to determine if it is a holiday or not.
+        return os.getenv('AG_COUNTRY', default='US')
 
     # TODO: Improve handling of missing datetimes
-    def _generate_features_datetime(self, X: DataFrame) -> DataFrame:
-        X_datetime = DataFrame(index=X.index)
+    def _generate_features_datetime(self, X: DataFrame, holiday_encoding=True, sincos_encoding=True) -> DataFrame:
```

Comment:
This ignores `self.holiday_encoding` and `self.sincos_encoding`, even if I set those values to False that wouldn't change the behavior.